paint_yourself_api/routes/styled_images.py

```diff
@@ -15,7 +15,7 @@
 def create_styled_image(
     *,
     _: FirebaseClaims = Depends(verify_authentication),
-    image_styler_service: ImageStylerService = Depends(ImageStylerService),
+    image_styler_service: ImageStylerService = Depends(get_image_styler_service),
```

Comment:
what's this `Depends`?

Reply:
`Depends` is from FastAPI which allows you do to dependency injection for routes.